pkg/kubelet/api/v1alpha1/lifecycle/api.proto

```diff
@@ -56,6 +56,7 @@ message Event {
 message EventReply {
   string error = 1;
   CgroupInfo cgroup_info = 2;
+  CgroupResource cgroup_resource = 3;
```

Comment:
@PiotrProkop now I understand what you were saying offline. Yep, agree this should probably be a repeated field. Want to do it as part of this PR?